packages/client/README.md

```diff
@@ -132,6 +132,30 @@ ethereumjs --mine --unlock=[ADDRESS1],[ADDRESS2],...
 
 Note that this feature is in `beta` and shouldn't be used with accounts holding a substantial amount of `Ether` on mainnet (or other valuable assets) for security reasons.
 
+### Custom network for development
+
+The client provides a quick way to get a local instance of the block chain up and running using the `--dev` command. This will start up a private PoA clique
+network with a prefunded account that also mines block on 10 second intervals. The prefunded account and its private key are printed to the screen when the client starts. When paired with the `--rpc` command, you have a ready-made environment for local development.
+
+```shell
+ethereumjs --dev --rpc
+
+==================================================
+Account generated for mining blocks:
+Address: 0xd8066d5822138e7c76d1565deb249f5f7ae370fa
+Private key: 0x6239e36ab8b27212868a1aa3f9c3b88b084075ea56aa4979d206371f065d3feb
+WARNING: Do not use this account for mainnet funds
+==================================================
+```
+
+Please **heed** the warning and do not use the provided account/private key for mainnet funds.
+
+This can also be paired with the `--unlock` command if you would like to specify the miner/prefunded account:
+```shell
+ethereumjs --dev --rpc --unlock=0xd8066d5822138e7c76d1565deb249f5f7ae370fa
```

Comment:
```suggestion
This can also be paired with the `--unlock` command if you would like to specify the miner/prefunded account:

```shell
ethereumjs --dev --rpc --unlock=0xd8066d5822138e7c76d1565deb249f5f7ae370fa
```